src/app/privacy-policy/page.tsx

```diff
@@ -0,0 +1,327 @@
+import React from "react";
+
+export default function privacypolicy() {
+  return (
+    <div className="mt-[4.5rem] flex w-full flex-col items-center justify-center">
+      <div className="w-full max-w-screen-2xl">
```

Comment:
@sudiksha255 Add some padding here maybe p-4? see if it looks good on your screen